File: scores.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int n = get_int("How many scores? ");

    int scores [n];

    for (int i = 0; i < n; i++)
    {
        scores[i] = get_int("Score: ");
    }
    printf("Average: %f\n", (scores[1] + scores[2] + scores[3]) / 3.0);
}

File: scores.py
from cs50 import get_int

scores = []

for i in range(3):
    score = get_int("Score: ")
    scores.append(score)
    #can also combine lists sample --->  scores = scores + [score]

average = sum(scores) / len(scores)

print(f"Average: {average:.2f}")